### PR TITLE
ACS-error-page 404 and 500

### PR DESCRIPTION
**Pull request by rakeshkumar-verma**
ACS-error-page 404 and 500

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/sling/servlet/errorhandler/404.jsp [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<%@page session="false"%><%
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/sling/servlet/errorhandler/404.jsp:1]: @rakeshkumar-verma : why are we including jsp pages here

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/config/com.adobe.acs.commons.errorpagehandler.impl.ErrorPageHandlerImpl.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OsgiConfig"
+    enabled="{Boolean}true"
+    error-page.fallback-name="500"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/config/com.adobe.acs.commons.errorpagehandler.impl.ErrorPageHandlerImpl.xml:5]: @rakeshkumar-verma : where have you configured 404 error ?

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@rakeshkumar-verma : pl check the chnages required

File: ui.apps/src/main/content/jcr_root/apps/sling/servlet/errorhandler/404.jsp (hunk 1 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/sling/servlet/errorhandler/404.jsp:1]: @rishabh-behl  this is default implementation as per acs.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/config/com.adobe.acs.commons.errorpagehandler.impl.ErrorPageHandlerImpl.xml (hunk 2 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/config/com.adobe.acs.commons.errorpagehandler.impl.ErrorPageHandlerImpl.xml:5]: @rishabh-behl  Based on error code corresponding error page will be picked from .../errorPages  page.

**Pull request merged**